server/host/host.cpp

```diff
@@ -0,0 +1,90 @@
+#include <openenclave/host.h>
+#include <stdio.h>
+
+#include "../enclave.h"
+
+// Include the untrusted modelaggregator header that is generated
+// during the build. This file is generated by calling the
+// sdk tool oeedger8r against the modelaggregator.edl file.
+#include "modelaggregator_u.h"
+
+char* path;
+uint32_t flags;
+
+bool check_simulate_opt(int* argc, const char* argv[])
+{
+    for (int i = 0; i < *argc; i++)
+    {
+        if (strcmp(argv[i], "--simulate") == 0)
+        {
+            fprintf(stdout, "Running in simulation mode\n");
+            memmove(&argv[i], &argv[i + 1], (*argc - i) * sizeof(char*));
+            (*argc)--;
+            return true;
+        }
+    }
+    return false;
+}
+
+// This is the function that the Python code will call into
+// Returns NULL on failure, new encrypted model on success
+unsigned char** host_modelaggregator(unsigned char*** encrypted_accumulator, 
+        uint32_t* decrypted_accumulator_lengths,
```

Comment:
I think prefixing the length variables with `decrypted_` is potentially misleading since their values were never encrypted to begin with. I think it's best to drop this prefix.